DOCS/man/vf.rst

```diff
@@ -750,6 +750,57 @@ Available mpv-only filters are:
     ``nvidia-true-hdr``
         Enable NVIDIA RTX Video HDR processing.
 
+``amf_frc``
+    AMD Frame Rate Conversion filter. Requires AMD hardware and drivers
+    supporting AMF FRC.
+
+    AMF FRC is a technique for achieving high-end video frame rate conversion
+    results from lower frame rate video inputs. The filter doubles the input
+    frame rate by generating intermediate frames.
+
+    AMF FRC only supports D3D11 input currently. Use `--hwdec=d3d11va` or
+    `--vf-pre=format=d3d11` to upload the data. `--d3d11va-zero-copy` is also
```

Comment:
It's very difficult for users to find out the solution to make `format=d3d11` work. After reading the source codes I knew a hwdec interop context is required...

Reply:
> I was unable to get --vf-pre=format=d3d11 working, is there something else I need to do to get that working?

```
 hwdec = no
 gpu-hwdec-interop = d3d11va
 vf=format = d3d11,amf_frc
```